tests/pipeline/test_adjusted_array.py

```diff
@@ -745,3 +745,42 @@ def test_update_labels(self):
         # values and adjustment values.
         check_arrays(adj_array.data, expected_adj_array.data)
         self.assertEqual(adj_array.adjustments, expected_adj_array.adjustments)
+
+    @parameterized.expand([
+        # Append new adjustment to back of the list if index already contains
+        # adjustments.
+        (
```

Comment:
This test is pretty readable because the inputs are small and the behavior under test is pretty simple, but IMO this usage of `parameterized` makes it harder to follow, because `parameterized` adds more space between the expectation and the assertion, and it puts space between the expectation and the assertion.

If I were writing this, I would probably structure it as something like:

```
@parameterized.expand([
    (initial_adjustments, added_adjustments, expected_result_when_prepending, expected_result_when_appending)
])
```

That structure would put the inputs and expected outputs closer together in the file, and would also make it easier to test more complex cases without making everything more complex.